### PR TITLE
Feature/kuz473 user rights documentation

### PR DESCRIPTION
**Pull request by benoitvidis**
Documentation for users/profiles/roles & role definition.

:arrow_right:  [Shorcut to the feature branch result](https://github.com/kuzzleio/kuzzle/tree/feature/kuz473-user-rights-documentation/docs/security)

### CODE REVIEW

**Comment by benoitvidis** [comment]
+1

**Comment by AnthonySendra** [comment]
+1
Very happy to see a documentation about security! :+1:

File: docs/README.md [hunk 1]
```diff
@@ -13,17 +13,17 @@ For the time being, Kuzzle implements the following protocols:
 * STOMP
 * Web Socket
 
-[Browse Kuzzle's API documentation](http://kuzzleio.github.io/kuzzle-api-documentation).
+Browser [Kuzzle's API documentation](http://kuzzleio.github.io/kuzzle-api-documentation).
```
Comment (dbengsch) [inline docs/README.md]: Browser ?

**Comment by dbengsch** [comment]
+1

**Comment by scottinet** [comment]
+1

**Pull request merged**